### ISSUE TITLE
Organization Dashboard UI

### ISSUE THREAD

**Issue by shreya024** [SrijanShovit/carbonops-v2#84]
## Description:
- Make UI for Org Dashboard after login/SignUp by an Org 
- Set the fields for Organization Profile Completion as the following
  - Contact No.
  - Location
  - Waste Requirements
  - Website
  - Social Media Links
  - Type of Organization
  - Product Deals
- Use React Mui/Next UI with Tailwind CSS

## Previous ER Diagram
![Organisation dashboard- for Organisation](https://github.com/SrijanShovit/carbonops-v2/assets/72400676/463a5c93-eaa9-4e5f-936b-39551e5695d6)

**Comment by SrijanShovit** [comment]
Hello shreya024, thanks for opening a issue, your contribution is valuable to us. The maintainers will review this issue and provide feedback as soon as possible.<p></p>

**Comment by MahendraDani** [comment]
@shreya024 I would like to work on this issue!

**Comment by MahendraDani** [comment]
@shreya024 Would you please also tell me the route for this page?
Thanks!

**Comment by MahendraDani** [comment]
@shreya024 
Should I add the (pencil icon) on each input field as given in the design?

**Comment by shreya024** [comment]
> @shreya024 
> Should I add the (pencil icon) on each input field as given in the design?
> And 
> Where should the Organization Profile Completion fields are to be added?
> 

Yes,the pencil icon is for editing the fields and profile completion fields should be inside Update Details

**Comment by MahendraDani** [comment]
@shreya024 Thanks!
So for now I need to just add the icons right? No need to work on their functionality right?

**Comment by shreya024** [comment]
> @shreya024 Thanks!
> So for now I need to just add the icons right? No need to work on their functionality right?

User should be able to edit the input fields on Clicking the pencil icons that's all

**Comment by MahendraDani** [comment]
Alright!

**Comment by MahendraDani** [comment]
@shreya024 Sorry it's taking a bit longer than usual! But will finish this asap!

**Comment by MahendraDani** [comment]
@shreya024 
Please reassign this issue to other contributor. I am not able to fix this.
Sorry!

**Comment by suzy-g38** [comment]
I would like to work on it.

**Comment by suzy-g38** [comment]
@shreya024 can you explain the issue a bit more clearly, I have started the work and am stuck in the middle. And also please confirm is it the demo UI or I have to do exactly whatever is shown in the image that you shared?

**Comment by shreya024** [comment]
> @shreya024 can you explain the issue a bit more clearly, I have started the work and am stuck in the middle. And also please confirm is it the demo UI or I have to do exactly whatever is shown in the image that you shared? 

Exact same api

**Comment by suzy-g38** [comment]
> > @shreya024 can you explain the issue a bit more clearly, I have started the work and am stuck in the middle. And also please confirm is it the demo UI or I have to do exactly whatever is shown in the image that you shared?
> 
> Exact same api

okay...I am having difficulty in understanding what the exact thing you want me to do,
1. I have to create an entire dashboard right?
2. if yes and as I am having difficult in understanding the given description, can I do this demo ui
3. 
![image](https://github.com/SrijanShovit/carbonops-v2/assets/62666287/25711ffd-cbea-49fd-a7a2-7910ee64ca86)

**Comment by shreya024** [comment]
> > > @shreya024 can you explain the issue a bit more clearly, I have started the work and am stuck in the middle. And also please confirm is it the demo UI or I have to do exactly whatever is shown in the image that you shared?
> > 
> > 
> > Exact same api
> 
> okay...I am having difficulty in understanding what the exact thing you want me to do,
> 
> 1. I have to create an entire dashboard right?
> 2. if yes and as I am having difficult in understanding the given description, can I do this demo ui
> 
> ![image](https://user-images.githubusercontent.com/62666287/248572473-25711ffd-cbea-49fd-a7a2-7910ee64ca86.png)

@suzy-g38 Yes the task is to create an entire dashboard based on the provided design.You can also view the other already merged UI issues or #104 for understanding/reference

**Comment by suzy-g38** [comment]
> > > > @shreya024 can you explain the issue a bit more clearly, I have started the work and am stuck in the middle. And also please confirm is it the demo UI or I have to do exactly whatever is shown in the image that you shared?
> > > 
> > > 
> > > Exact same api
> > 
> > 
> > okay...I am having difficulty in understanding what the exact thing you want me to do,
> > 
> > 1. I have to create an entire dashboard right?
> > 2. if yes and as I am having difficult in understanding the given description, can I do this demo ui
> > 
> > ![image](https://user-images.githubusercontent.com/62666287/248572473-25711ffd-cbea-49fd-a7a2-7910ee64ca86.png)
> 
> @suzy-g38 Yes the task is to create an entire dashboard based on the provided design.You can also view the other already merged UI issues or #104 for understanding/reference

Thanks @shreya024....I will check that.
